src/app/common/modal/container-modal/container-modal.component.scss

```diff
@@ -0,0 +1,12 @@
+@import '../../../../styles/ngx-bootstrap/modal';
```

Comment:
This should not be imported here.  It will result in duplicate style definitions in the final build.

Reply:
Thanks @jrassa ! Is import the `styles/shared` the right way to go?

Reply:
That works.  Generally, in the component styles, we only want to import files that define variables/functions/mixins to avoid duplicated styles.

Reply:
I reduced the scope of this import further from `styles/shared` to `styles/colors`, since those were the only variables we needed here. Thanks again @jrassa  !